core/src/main/scala/dagr/core/tasksystem/Pipe.scala

```diff
@@ -80,6 +80,13 @@ trait Pipe[In,Out] extends ProcessTask {
   private[tasksystem] def ordered: Seq[Pipe[_,_]] = Seq[Pipe[_,_]](this)
 }
 
+/** A trait for pipe tasks that consumes no cores and no memory */
+trait PipeWithNoResources[In,Out] extends Pipe[In,Out] with FixedResources {
```

Comment:
I think the implementation of this is fine, but I feel a bit weird about having this in `tasksystem` since it's basically a utility trait that extends two other traits.  That said, I'm not sure there's a better place for it.